docs/reference/http-endpoints.md

```diff
@@ -0,0 +1,190 @@
+# HTTP API endpoint list
+
+Here is the full list for the various HTTP paths and their usage in GreptimeDB:
+
+## Admin APIs
+
+Endpoints that is not versioned (under `/v1`). For admin usage like health check, status, metrics, etc.
+
+### Health Check
+
+- **Path**: `/health`
+- **Methods**: `GET`, `POST`
+- **Description**: Provides a health check endpoint to verify that the server is running.
+- **Usage**: Access this endpoint to check the health status of the server.
+
+### Status
+
+- **Path**: `/status`
+- **Methods**: `GET`
+- **Description**: Retrieves the current status of the server.
+- **Usage**: Use this endpoint to obtain server status information.
+
+### Metrics
+
+- **Path**: `/metrics`
+- **Methods**: `GET`
+- **Description**: Exposes Prometheus metrics for monitoring purposes.
+- **Usage**: Prometheus can scrape this endpoint to collect metrics data.
+
+### Configuration
+
+- **Path**: `/config`
+- **Methods**: `GET`
+- **Description**: Retrieves the server's configuration options.
+- **Usage**: Access this endpoint to get configuration details.
+
+### Dashboard
+
+- **Paths**: `/dashboard`
+- **Methods**: `GET`, `POST`
+- **Description**: Provides access to the server's dashboard interface.
+- **Usage**: Access these endpoints to interact with the web-based dashboard.
+
+This dashboard is packaged with the GreptimeDB server and provides a user-friendly interface for interacting with the server. It requires corresponding compile flags to be enabled when building GreptimeDB. The original source code for the dashboard can be found at https://github.com/GreptimeTeam/dashboard
+
+### Debugging Tools
+
+- **Base Path**: `/debug`
+- **Endpoints**:
+  - `/log_level`
+  - `/pprof/*`
+- **Methods**:
+  - `POST` for changing log levels.
+  - `GET` for profiling endpoints.
+- **Description**: Offers debugging and profiling tools.
+- **Usage**:
+  - Use `/log_level` to adjust the server's log level dynamically.
+  - Use `/pprof/*` endpoints for performance profiling.
+
+## Query Endpoints
+
+Various query APIs for sending query to GreptimeDB.
+
+### SQL API
+
+- **Path**: `/v1/sql`
+- **Methods**: `GET`, `POST`
+- **Description**: Executes SQL queries against the server.
+- **Usage**:
+  - `GET`: Send SQL queries via query parameters.
```

Comment:
```suggestion
```
As discussed before, We do not recommend use the GET method to query data. All SQL API should use `POST`, which means post SQL statements according to the RESTful Principles.